src/main/scala/io/iohk/ethereum/transactions/PendingTransactionsManager.scala

```diff
@@ -54,9 +54,13 @@ class PendingTransactionsManager(txPoolConfig: TxPoolConfig, peerManager: ActorR
 
   implicit val timeout = Timeout(3.seconds)
 
-  peerMessageBus ! Subscribe(MessageClassifier(Set(SignedTransactions.code), PeerSelector.AllPeers))
+  peerEventBus ! Subscribe(SubscriptionClassifier.PeerHandshaked)
+  peerEventBus ! Subscribe(MessageClassifier(Set(SignedTransactions.code), PeerSelector.AllPeers))
 
   override def receive: Receive = {
+    case PeerEvent.PeerHandshakeSuccessful(peer, _) =>
+      self ! NotifyPeer(pendingTransactions.map(_.stx), peer)
```

Comment:
yes, but as I said, it may not be worth implementing because we already have limit for the pool